.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,39 +1,57 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: "\U0001F41E Bug report"
+about: Report an issue with Vite
 title: ''
 labels: 'bug: pending triage'
 assignees: ''
 ---
 
-<!--
-中文用户请注意：请仔细阅读以下模版，如果不遵循模版，issue 将会被直接关闭。
--->
-
-**⚠️ IMPORTANT ⚠️ Please do not ignore this template. If you do, your issue will be closed immediately.**
+<!-- 中文用户请注意：请仔细阅读以下模版，如果不遵循模版，issue 将会被直接关闭。 -->
+<!-- IMPORTANT: Please do not ignore this template. If you do, your issue will be closed immediately. -->
```

Comment:
The ⚠️ emoji really draws attention to the warning, versus the newly commented out version. Maybe add that bad in?

Reply:
As the same with https://github.com/vitejs/vite/pull/2645#discussion_r598971856, the text is only visible in the draft view of the issue and shown in the comment, therefore it is not rendered as an icon/emoji but an unicode symbol.

Is there maybe another way of drawing attention?

Maybe we could try `<!-- !!! IMPORTANT !!! Please do not ignore this template. If you do, your issue will be closed immediately. -->`